internal/plugin/backupplugin.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package plugin
```

Comment:
Do we not want to separate the different type of plugins into their respective packages? Grouping backup item action plugins into `backup` package...
We did that in the CSI plugin repo. Following the same here, we'll have a plugin per package.

Reply:
I can do that if you all prefer; it seemed unnecessary here since there was only one plugin per type, but i'm fine either way.

Reply:
I agree that it is unnecessary in this case. I am OK w/ all plugins being in the same package.

Reply:
always easy to change down the road too, if we add more examples.

Reply:
sgtm!